moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -212,47 +235,84 @@ struct FCLGeometry
     collision_geometry_->setUserData(collision_geometry_data_.get());
   }
 
+  /** \brief Pointer to FCL collision geometry. */
   std::shared_ptr<fcl::CollisionGeometryd> collision_geometry_;
+
+  /** \brief Pointer to the user defined geometry data. */
   CollisionGeometryDataPtr collision_geometry_data_;
 };
 
 typedef std::shared_ptr<fcl::CollisionObjectd> FCLCollisionObjectPtr;
 typedef std::shared_ptr<const fcl::CollisionObjectd> FCLCollisionObjectConstPtr;
 
+/** \brief A general hight level object which consists of multiple \e FCLCollisionObjects. It is the top level data
+ *  structure which is used in the collision checking process. */
 struct FCLObject
 {
   void registerTo(fcl::BroadPhaseCollisionManagerd* manager);
   void unregisterFrom(fcl::BroadPhaseCollisionManagerd* manager);
   void clear();
 
   std::vector<FCLCollisionObjectPtr> collision_objects_;
+
+  /** \brief To \e collision_objects_ corresponding geometry data. */
   std::vector<FCLGeometryConstPtr> collision_geometry_;
 };
 
+/** \brief Bundles an \e FCLObject and and a broadphase FCL collision manager. */
 struct FCLManager
 {
   FCLObject object_;
   std::shared_ptr<fcl::BroadPhaseCollisionManagerd> manager_;
 };
 
+/** \brief Callback function used by the FCLManager used for each pair of collision objects to
+*   calculate object contact information.
+*
+*   \param o1 First FCL collision object
+*   \param o2 Second FCL collision object
+*   \data General pointer to arbitrary data which is used during the callback
+*   \return True terminates the distance check, false let it continue to the next pair of objects */
 bool collisionCallback(fcl::CollisionObjectd* o1, fcl::CollisionObjectd* o2, void* data);
 
+/** \brief Callback function used by the FCLManager used for each pair of collision objects to
+*   calculate collisions and distances.
+*
+*   \param o1 First FCL collision object
+*   \param o2 Second FCL collision object
+*   \data General pointer to arbitrary data which is used during the callback
+*   \return True terminates the collision check, false let it continue to the next pair of objects */
```

Comment:
```suggestion
*   \return True terminates the collision check, false lets it continue to the next pair of objects */
```

Reply:
Changed it to 
 > True terminates the collision check, false continues it to the next pair of objects